haskell/test/Test/Ligo/BaseDAO/Proposal/Propose.hs

```diff
@@ -0,0 +1,1119 @@
+-- SPDX-FileCopyrightText: 2021 TQ Tezos
+-- SPDX-License-Identifier: LicenseRef-MIT-TQ
+
+-- | Contains tests on @propose@ entrypoin logic for testing the Ligo contract.
```

Comment:
For `Management` I think the change is fine, but regarding `Proposal` and this module, isn't this now the big one with the same very long compilation time?

I think the scenarios here could be moved to different submodules, e.g. at first glance these could be created:
- `Test.Ligo.BaseDAO.Proposal.Quorum` for tests only about the quorum to `propose`
- `Test.Ligo.BaseDAO.Proposal.Tokens` for tests only about the frozen tokens conditions to respect
- `Test.Ligo.BaseDAO.Proposal.Vote` for tests on `vote` only
- etc.

I think this would make it easier to also check that everything is tested and find tests.
e.g. if a condition for `vote` is changed I know that I should add/update a test in `Test.Ligo.BaseDAO.Proposal.Vote`

What do you think?

Reply:
I have split them into more modules  and have added new modules`Flush`, `Quorum`, `Token` and moved some tests from `Propose` to `Vote` module.